### PR TITLE
Initial other-checks benchmark

### PR DESCRIPTION
**Pull request by rajlearner17**
### Checklist
- [ ] Issue(s) linked

### CODE REVIEW

File: query/elb/elb_application_classic_network_lb_prohibit_public_access.sql [hunk 1]
```diff
@@ -0,0 +1,44 @@
+with all_lb_details as (
+ 	select
+		arn,
```
Comment (cbruno10) [inline query/elb/elb_application_classic_network_lb_prohibit_public_access.sql:3]: @rajlearner17 Why is this query using tabs (along with others)?

File: other_checks/other.sp [hunk 2]
```diff
@@ -0,0 +1,73 @@
+locals {
+  other_common_tags = merge(local.aws_compliance_common_tags, {
+    other = "true"
+  })
+}
+
+benchmark "other" {
+  title         = "Other"
```
Comment (cbruno10) [inline other_checks/other.sp:8]:
```suggestion
  title         = "Other Compliance Checks"
```

**Review by cbruno10 — CHANGES_REQUESTED** [review]
@rajlearner17 I know this PR is still in draft, but I took an initial look as I know we're releasing this week. Please see comments, if any are already changed, please disregard, thanks!

File: query/eks/eks_cluster_control_plane_audit_logging_enabled.sql [hunk 3]
```diff
@@ -0,0 +1,31 @@
+with control_panel_audit_logging as (
+  select
+    distinct arn,
+    log -> 'Types'  as log_type
```
Comment (misraved) [inline query/eks/eks_cluster_control_plane_audit_logging_enabled.sql:4]:
```suggestion
    log -> 'Types' as log_type
```

File: query/eks/eks_cluster_control_plane_audit_logging_enabled.sql [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+  from
+    aws_eks_cluster,
+    jsonb_array_elements(logging -> 'ClusterLogging') as log
+  where
+    log ->> 'Enabled' = 'true'
+    and (log -> 'Types') @> '["api","audit", "authenticator", "controllerManager","scheduler"]'
```
Comment (misraved) [inline query/eks/eks_cluster_control_plane_audit_logging_enabled.sql:10]:
```suggestion
    and (log -> 'Types') @> '["api", "audit", "authenticator", "controllerManager", "scheduler"]'
```

File: query/elb/elb_application_classic_lb_with_outbound_rule.sql [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+  distinct l.arn as resource,
+  case
+    when l.security_groups is null then 'alarm'
+    when s.num = 0 then 'alarm'
+    else 'ok'
+  end as status,
+  case
+    when l.security_groups is null then l.title || ' does not have security group attached.'
+    when s.num = 0 then l.title || ' does not have outbound rule.'
+    else l.title || ' have outbound rule(s).'
```
Comment (misraved) [inline query/elb/elb_application_classic_lb_with_outbound_rule.sql:56]: Can we add how many outbound rules are present?

File: query/elb/elb_listener_use_secure_ssl_cipher.sql [hunk 6]
```diff
@@ -0,0 +1,20 @@
+select
+ -- Required Columns
```
Comment (misraved) [inline query/elb/elb_listener_use_secure_ssl_cipher.sql:2]:
```suggestion
  -- Required Columns
```

File: query/glue/glue_data_catalog_encryption_settings_password_encryption_enabled.sql [hunk 7]
```diff
@@ -0,0 +1,15 @@
+select
+  -- Required Columns
+  case
+    when connection_password_encryption is not null and connection_password_encryption ->> 'ReturnConnectionPasswordEncrypted' != 'false' then 'ok'
+    else 'alarm'
+  end as "status",
+  case
+    when connection_password_encryption is not null and connection_password_encryption ->> 'ReturnConnectionPasswordEncrypted' != 'false' then 'enabled glue data catalog connection password encryption in ' || region
```
Comment (misraved) [inline query/glue/glue_data_catalog_encryption_settings_password_encryption_enabled.sql:8]: Please make changes as per the above suggestion

File: query/glue/glue_dev_endpoint_s3_encryption_enabled.sql [hunk 8]
```diff
@@ -0,0 +1,18 @@
+select
+  -- Required Columns
+  d.arn as resource,
+  case
+    when e is not null and e ->> 'S3EncryptionMode' != 'DISABLED' then 'ok'
+    else 'alarm'
+  end as "status",
+  case
+    when e is not null and e ->> 'S3EncryptionMode' != 'DISABLED' then d.title || ' s3 encryption enabled.'
```
Comment (misraved) [inline query/glue/glue_dev_endpoint_s3_encryption_enabled.sql:9]:
```suggestion
    when e is not null and e ->> 'S3EncryptionMode' != 'DISABLED' then d.title || ' S3 encryption enabled.'
```

File: query/glue/glue_dev_endpoint_s3_encryption_enabled.sql [hunk 9]
```diff
@@ -0,0 +1,18 @@
...
+    else d.title || ' s3 encryption disabled.'
```
Comment (misraved) [inline query/glue/glue_dev_endpoint_s3_encryption_enabled.sql:10]:
```suggestion
    else d.title || ' S3 encryption disabled.'
```

File: query/iam/iam_policy_custom_no_assume_role.sql [hunk 10]
```diff
@@ -0,0 +1,30 @@
...
+)
+select
+  -- Required Columns
+  u.arn as resource,
+  case
+    when fu.user_id is not null then 'alarm'
+    else 'ok'
+  end as status,
+  case
+    when fu.user_id is not null then u.name || ' custom policies allows STS Role assumption.'
```
Comment (misraved) [inline query/iam/iam_policy_custom_no_assume_role.sql:20]:
```suggestion
    when fu.user_id is not null then u.name || ' custom policies allow STS Role assumption.'
```

File: query/iam/iam_user_with_administrator_access_mfa_enabled.sql [hunk 11]
```diff
@@ -0,0 +1,32 @@
...
+  -- Required Columns
+  u.arn as resource,
+  case
+    when au.user_id is null then 'skip'
+    when au.user_id is not null and u.mfa_enabled then 'ok'
+    else 'alarm'
+  end as status,
+  case
+    when au.user_id is null then u.name || ' does not have administrator access.'
+    when au.user_id is not null and u.mfa_enabled then u.name || ' have MFA token enabled.'
```
Comment (misraved) [inline query/iam/iam_user_with_administrator_access_mfa_enabled.sql:22]:
```suggestion
    when au.user_id is not null and u.mfa_enabled then u.name || ' has MFA token enabled.'
```

**Pull request merged**